spec/classes/plugin_deb_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe 'pulpcore::plugin::deb' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+
+      it { is_expected.to compile.with_all_deps }
+      it { is_expected.to contain_pulpcore__plugin('deb') }
```

Comment:
```suggestion
      it { is_expected.to compile.with_all_deps }
      it { is_expected.to contain_pulpcore__plugin('deb') }
      it { is_expected.to contain_package('python3-pulp-deb') }
```

To keep up with the changes in https://github.com/theforeman/puppet-pulpcore/commit/b70670825cc99f1aab28f9cb3192a00ad461c646

@m-bucher could you add this please before I merge ?